.platform/solr_config/solrcore.properties.old

```diff
@@ -0,0 +1,13 @@
+solr.replication.master=false
```

Comment:
Not sure we need this .old file? Could well be autogenerated as a fallback, if needed.